pkg/querier/tail.go

```diff
@@ -106,6 +114,7 @@ func (t *Tailer) loop() {
 				continue
 			}
 
+			entriesSize += len(t.currEntry.Line)
```

Comment:
This isn't necessarily the number of bytes, but it is consistent with how we calculate similar metrics in other components.

Reply:
You're right: Golang `len` gives back bytes rather characters. I can't believe I didn't know this.

I wonder how many bugs I've caused with it?

Reply:
haha, go was clearly designed for dummies like me who only use English characters and still think ASCII is the only encoding out there.